styles.css

```diff
@@ -1,15 +1,23 @@
 .background {
-    background-color: #9E9FA5;
+    background-color: #ffe1cd;
```

Comment:
I changed the body color to a shade of grey and text to white.
Since you have changed the body color back to pink and text is still white, the text is not clearly visible at home page and contact us page.
Could you please keep the body color to grey(#9E9FA5) and change the header color accordingly.